src/responsive-grid.scss

```diff
@@ -0,0 +1,201 @@
+$spacer: 4px !default;
+
+$grid-gutter: $spacer * 6 !default;
+$grid-columns: 12;
+$grid-breakpoints: ('xs': 0,
+	'sm': 600px,
+	'md': 960px,
+	'lg': 1280px - 16px,
+	'xl': 1920px - 16px);
+
+$grid-gutters: ('xs': $grid-gutter / 12,
+	'sm': $grid-gutter / 6,
+	'md': $grid-gutter / 3,
+	'lg': $grid-gutter * 2/3,
+	'xl': $grid-gutter);
+
+@each $name,
+$glyph in $grid-breakpoints {
+	.icon-#{$name}:before {
+		display: inline-block;
+		font-family: "Icon Font";
+		content: $glyph;
+	}
+}
+
+*,
```

Comment:
Out of curiosity: Why are we using icons in a grid system?

Reply:
Well it was a test that i did using my [svg repo](https://github.com/LeandrodeLimaC/Svg-Render) sorry, i will remove those lines!